core/store/src/migrations.rs

```diff
@@ -186,3 +188,64 @@ pub fn migrate_31_to_32(storage: &crate::NodeStorage) -> anyhow::Result<()> {
     update.commit()?;
     Ok(())
 }
+
+fn get_row_count(store: &Store, col_name: &str) -> Option<i64> {
+    let statistics = store.get_store_statistics()?;
+    let col_stats = statistics.data.into_iter().find(|(col, _)| col == col_name)?.1;
+    for stat in col_stats {
+        match stat {
+            crate::db::StatsValue::Count(count) => return Some(count),
+            _ => {}
+        }
+    }
+    None
+}
+
+/// Migrates database from version 32 to 33.
+///
+/// This removes the TransactionResult column and moves it to TransactionResultForBlock.
+/// The new column removes the need for high-latency read-modify-write operations when committing
+/// new blocks.
+pub fn migrate_32_to_33(storage: &crate::NodeStorage) -> anyhow::Result<()> {
+    const BATCH_SIZE: usize = 100000;
+    let store = storage.get_store(crate::Temperature::Hot);
+    let mut migrated_overall = 0;
+    let num_rows = get_row_count(&store, "col7");
```

Comment:
Just use `BatchedStoreUpdate`.  It’s what it’s for.

Reply:
Updated to use that.